src/utils/durations.ts

```diff
@@ -0,0 +1,5 @@
+export const ONE_SECOND_MS = 1000;
+export const ONE_MINUTE_MS = 60 * ONE_SECOND_MS;
+export const ONE_HOUR_MS = 60 * ONE_MINUTE_MS;
```

Comment:
Will there be a follow-up PR to use this as source for `ONE_HOUR_MS` everywhere?
<img width="418" alt="Skjermbilde 2024-08-09 kl  13 20 05" src="https://github.com/user-attachments/assets/8e211730-05eb-4c68-ad33-af8c96fc7997">